Plugins/Vorlon/plugins/interactiveConsole/vorlon.interactiveConsole.dashboard.ts

```diff
@@ -313,7 +313,8 @@
             this.entry = entry;
             this.element = document.createElement("div");
             this.element.className = 'log-entry ' + this.getTypeClass();
-            parent.insertBefore(this.element, parent.childNodes.length > 0 ? parent.childNodes[0] : null);
+            this.element.title = "Message received at " + new Date().toLocaleTimeString();
+            parent.appendChild(this.element);
 
```

Comment:
Need to ``this.element.scrollIntoView()`` here otherwise its very hard to watch the console.  This also probably means that there needs to be an auto-scroll toggle for the console so the developer can disable the auto-scrolling when looking back through the log.  That's how chrome dev tools does it.